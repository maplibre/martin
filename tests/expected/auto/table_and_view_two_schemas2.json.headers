content-encoding: br
content-type: application/json
etag: W/"186-uyLSlK6Tfo32Ic8Ra_Q7JQ=="
transfer-encoding: chunked
vary: accept-encoding, Origin, Access-Control-Request-Method, Access-Control-Request-Headers
