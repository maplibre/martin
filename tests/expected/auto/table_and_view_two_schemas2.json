{
  "bounds": [
    0.0012741833,
    9.88181e-05,
    0.0012921496,
    0.0001169055
  ],
  "description": "schema_b.table_and_view_two_schemas.b_geom",
  "name": "table_and_view_two_schemas.1",
  "tilejson": "3.0.0",
  "tiles": [
    "http://localhost:3111/table_and_view_two_schemas.1/{z}/{x}/{y}"
  ],
  "vector_layers": [
    {
      "fields": {
        "b_id": "int4",
        "b_info": "text"
      },
      "id": "table_and_view_two_schemas.1"
    }
  ]
}
